### ISSUE TITLE
G6.0.12 - Link from pinnened tab opens left from normal tabs

### ISSUE THREAD

**Issue by TheDutchJewel** [MrAlex94/Waterfox#3411]
### What happened?

In the General Settings > Additional Tab Preferences, I have _disabled_ the options:
- Insert new tab after current tab
- Insert related new tab after current tab

I have a lot of pinned tabs open by default in Waterfox.

When I click on a _pinned_ tab to open it in a new tab (Ctrl+click), that tab always opened to the _right_ of the regular tabs.

But in the new version (G6.0.12), that tab opens to the _left_ of the regular tabs.

What should I do to open new links of pinned tabs to the _right_ of opened tabs?

PS: Have gone back to the previous version for now until there is a solution for this.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

G6

### What platform are you seeing the problem on?

Windows

**Comment by TheDutchJewel** [comment]
Same problem with G.0.13.

**Comment by MrAlex94** [comment]
Very strange, I can't reproduce this. Do you have a list of add-ons installed? Also, does the issue occur in [troubleshoot mode](https://support.mozilla.org/en-US/kb/troubleshoot-and-diagnose-firefox-problems#w_3-restart-firefox-in-troubleshoot-mode)?

**Comment by TheDutchJewel** [comment]
Yes, the problem occurs also in the [troubleshooting mode](https://support.mozilla.org/en-US/kb/troubleshoot-and-diagnose-firefox-problems#w_3-restart-firefox-in-troubleshoot-mode).

Until G.6.0.11, I used the extension `Tree Style Tab`. When updating to G6.0.12, Waterfox detected this extension and gives a warning at startup: `Conflicting Features Detected`:
![afbeelding](https://github.com/BrowserWorks/Waterfox/assets/46184729/e2776156-51a0-4c24-ae83-e08f0eee73ba)

When I select `Waterfox's Tabs Sidebar` option, it opens urls in new tabs on the **left**.

When I re-enable the Tree Style Tab extension I get the warning again. If I then select the `Tree Style Tab` option, I get another warning:
![afbeelding](https://github.com/BrowserWorks/Waterfox/assets/46184729/029a2401-8c0f-4b87-8864-f579bdce9e0b)
and Waterfox opens urls on the **right** side again.

So the problem occurs only when `Waterfox's Side Tab` is enabled.

**Comment by piroor** [comment]
This is a side effect of Waterfox Sidebar: Advanced Options (TST's options page, shift-click on the "More Settings") => (Expert options) => "Tabs opened from Existing Tabs" => "Insertion position of new child tabs from pinned tabs (will appear as root tabs)" => "Next to the recently opened child, or near the opener (simulates the browser's default behavior)". This option need to be switched to "The end of the tree" automatically, when both "Insert new tab after current tab" and "Insert related new tab after current tab" are unchecked by the user.
